src/codex/account-store.ts

```
@@ -321,58 +324,202 @@ function errCode(err: unknown): string | undefined {
+  try {
+    if (!refreshLockIsStale(path)) return;
+    // Acquirers hold the reclaim lock while creating the lock file, so this
+    // rename can only retire the stale entry that was just inspected.
+    renameSync(path, retiredPath);
+  } catch (error) {
+    if (errCode(error) !== "ENOENT") throw error;
+  } finally {
+    try {
+      unlinkSync(retiredPath);
+    } catch (error) {
+      if (errCode(error) !== "ENOENT") throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)